bridge/setu/util/common.go

```diff
@@ -476,3 +479,30 @@ func GetUnconfirmedTxnCount() int {
 
 	return count
 }
+
+// LogElapsedTimeForStateSyncedEvent logs useful info for StateSynced events
+func LogElapsedTimeForStateSyncedEvent(event interface{}, functionName string, startTime time.Time) {
+	switch event.(type) {
+	case statesender.StatesenderStateSynced:
+		event, ok := event.(statesender.StatesenderStateSynced)
+		if !ok {
+			logger.Error("Unable to typecast event to statesender.StatesenderStateSynced")
+			break
+		}
+		logger.Info("StateSyncedEvent: "+functionName,
+			"stateSyncId", event.Id,
```

Comment:
We just need an ID here. Could we then pass everywhere not an event type, but rather eventID?

Reply:
Same as other comment, if I am not mistaken I'd need to check if the type is `statesender.StatesenderStateSynced` in many other places and then typecast it to that event to extract the ID